### PR TITLE
List actions rendered as menu on larger screens

### LINKED ISSUE #2790: [ENHANCEMENT] Item: Swipe actions for web (non-touch)
- [ X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the solution you'd like
Swipe actions for web (non-touch)
* On web swipe actions are displayed in an (additional?) item segment like an item with "07 Icon" in the end segment
* If only one action is available, the corresponding icon can be displayed in the segment (if the swipe action can be associated with an appropriate icon)
* If no icon is specified for a single swipe action a 01 Icons/general/more icon will be displayed. Pressing the icon will invoke an action list (dropdown) with the action name (appropriate action names must be specified for alle swipe actions). Pressing the action name will invoke the action.
* If more than one swipe action is available a 01 Icons/general/more icon will be displayed. Pressing the icon will invoke an action list (dropdown) with alle the action names (appropriate action names must be specified for alle swipe actions). Pressing an action name will invoke the corresponding action.

## Have you considered any alternatives?
Nope

## Are there any additional context?
image (8)

ZEPLIN: [https://zpl.io/7wnwwej](url)

Any questions please reach out to Lars Bruun: lab@bankdata.dk

(I am only a puppet creating this issue due to Lars being UX and thus not having a github account)

### PR DESCRIPTION
**Pull request by kodeaben**
## Which issue does this PR close?

This PR closes #2790 
## What is the new behavior?
When the device is a desktop, and the list item has actions. A menu will appear at the far right.
I there is only one action, the action itself will be shown insead of the menu.
When clicking said menu, the actions will be displayed.
If the swipeactions are disabled, the menu/action button will be disabled as well.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

**Comment by kodeaben** [comment]
ListSwipeSpec

File: libs/designsystem/list/src/list.component.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <ion-list
+  id="kirbyListPortalOutletHook"
```
Comment (kodeaben) [inline libs/designsystem/list/src/list.component.html:2]: might be solved by menu defaulting to document

File: libs/designsystem/list/src/list.component.scss [hunk 2]
```diff
@@ -168,7 +168,7 @@ ion-item-options[side='end'] {
   }
 
   ion-item.first,
-  ion-item-sliding.first,
+  kirby-list-item.first,
```
Comment (kodeaben) [inline libs/designsystem/list/src/list.component.scss:171]: triple double check that this change works on all devices and browsers

**Comment by RasmusKjeldgaard** [comment]
TODO

**Cookbook**
- [ ] Rename cookbook examples to reflect that swipe actions are now actions
 
**Implementation**
- [ ] Refactor`list-item.component.html` a bit to read better with `ng-templates` at the bottom of the file not 'slotted in' in the context

File: apps/cookbook/src/app/examples/menu-example/examples/portal.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { ChangeDetectorRef, Component, Input } from '@angular/core';
 const config = {
   selector: 'cookbook-menu-portal-example',
   template: `<kirby-menu 
+  [isDisabled]="true"
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/examples/menu-example/examples/portal.ts]: Make sure isDisabled flag on menu is forwarded to the slotted button

**Pull request closed without merging**